docs/user-research/MO-user-groups-feedback.md

```diff
@@ -0,0 +1,276 @@
+  responded 'yes'.] So no upgrade process will be required? ['Correct'.] Will
+  anything be deprecated? [We said maybe but emphasised that we would make
+  sure users are informed of any deprecations.]
+
+* RR: A new UI is good, but I actually prefer the command line, e.g. because
+  I can explicitly see & use the whole "plehtora of options". Will I still
+  be able to interact with my suites purely by command, as I can do now? [We
+  said yes, but stressed that we are not neglecting the CLI for the new UI,
+  in fact we have plans including a wide-ranging "CLI refactor" Issue
+  (``cylc-flow``, #2972) open for Cylc 8, & towards that have already made
+  various improvements to the command line].
+
+* RR: When you interact with a suite via the UI, it would be very useful to
+  see the corresponding commands, e.g. in command logs (obviously there will be
+  no command equivalent to the action of 'adding in views' etc., but for
+  anything with a suite command equivalent). [We agreed that it would indeed be
+  useful & we would note it, but also described how back-end changes mean that
+  the UI & command line will be synchronised so that most/all commands
+  correspond to UI actions & vice versa, & the command equivalents should be
+  mostly evident.]
+
+* AA: How will this all work for shared accounts? [We explained we are still
+  planning what to do with shared accounts, but probably the shared account
+  owner will grant various levels of access (e.g. limited control, read-only,
+  etc.) to sets of users as desired, such that when logged-in to the Cylc Hub
+  as themselves, users will be able to interact, to the permitted extent, with
+  any shared accounts which owners have allowed them to.]
+
+* MT: What are those panels containing the views? [We elaborated on
+  the multi-view system in the design mock-up.]
+
+* MB: Will you still be able to have multiple instances open, like multiple
+  windows for the old GUIs? I guess it would be multiple tabs now? [We
+  confirmed that multiple tabs would be the browser equivalent, & that the UI
+  will work across mutliple tabs, but explained how we are trying to reduce
+  the need for multiple (and especially large numbers of) tabs with the
+  multi-view panelling system, which aims to show users as much information
+  as they could want to see, all within a single tab.
+
+* AD: We have set up a feedback page under the retooling project pages & we'll
+  keep putting recommendations on there [We said thanks.].
+
+* MB: In general, Cylc meets our needs, but Rose does not. [We said this was
+  noted.]
+
+
+#### From feedback pages
+
+Notable points (relating to the Cylc UI) from feedback pages:
+
+* [From the agenda notes:] "TSUG noted that this [the Rose replacement] will
+  be of most benefit to users represented here. The *Cylc upgrades are nice
+  to have* but the Rose upgrades will be vital to allow us to get the most
+  out of our trials."
```

Comment:
Huh.  Well I would emphasise to these users that the "upgrade" aspect of Cylc 8 will be nice but is actually something of a side-effect. The main thing is: *we **have to do this** because Python 2 and PyGTK are **dead**!*

Reply:
Very true, though sadly I don't think (from my recollection) that we managed to convey that, mainly because we only had a short time slot to discuss feedback & that the mention of Rose gave us a cue to emphasise, as we thought important, that we did want to hear feedback on Rose (just not in that particular session, which was devoted to the Cylc UI). And then discussion flowed from that such that the topic changed before we could describe the requirement to "upgrade".

Hopefully it was somewhat evident from context, though, that there were push factor(s) & we weren't just changing the UI etc. because it would be nice to do.